packages/auth/src/config/app.ts

```diff
@@ -25,13 +25,13 @@ export const Config = {
         ),
   authServerSpec: envString(
     'AUTH_SERVER_SPEC',
-    // TODO: replace with commit on main once it's merged in
-    'https://raw.githubusercontent.com/interledger/open-payments/0551bb8929931750e298ae177f6459118d8936e3/auth-server-open-api-spec.yaml'
+    'https://raw.githubusercontent.com/interledger/open-payments/53236cc92a070ec98a7ab31c6ed0f6fc50a98041/auth-server-open-api-spec.yaml'
   ),
   identityServerDomain: envString(
     'IDENTITY_SERVER_DOMAIN',
     'http://localhost:3004'
   ),
+  identityServerSecret: envString('IDENTITY_SERVER_SECRET', 'replace-me'),
```

Comment:
@njlie can we add an issue to audit and confirm the security of how we doing the secrets here please.

Reply:
Captured in https://github.com/interledger/rafiki/issues/563